mmrazor/models/architectures/connectors/ofd_connector.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, Optional
+
+import torch
+
+from mmrazor.registry import MODELS
+from .base_connector import BaseConnector
+
+
+@MODELS.register_module()
+class OFDTeacherConnector(BaseConnector):
```

Comment:
UT

Reply:
OFD's new components including `OverhaulFeatureDistillation`, `OFDDistiller` and `OFDTeacherConnector` should be tested as whole. Skipping coding individual UTs